tests/YGMeasureTest.cpp

```diff
@@ -21,8 +21,24 @@ static YGSize _measure(YGNodeRef node,
   }
 
   return YGSize {
-      .width = 10,
-      .height = 10,
+      width = 10,
```

Comment:
Well the syntax is not part of the c99 designated initializer syntax as far as I know. I would rather stick to the standard for now.